Tests/AttributedStringStyleTests.swift

```diff
@@ -215,17 +215,17 @@ class AttributedStringStyleTests: XCTestCase {
     //   - an empty style object that is updated by the passed style object
     //   - a fully populated style object that is updated by the passed style object
     func checks(for style: AttributedStringStyle) -> [(style: AttributedStringStyle, fullStyle: Bool)] {
-        let derived = AttributedStringStyle.style().derive { derived in
+        let derivedStyle = AttributedStringStyle.style().byAdding { derivedStyle in
             // Not sure why but if this line is commented out, the line after is a compilation error.
```

Comment:
To what? It's still a derived style. Or do you want to just strip out the block and do this one inline?